docs/running-node.md

```diff
@@ -150,40 +167,53 @@ Run
 $ ./run_dcl_node -t genesis -c <chain-id> node0
 ```
 
+*Notes*
+
+*   the script assumes that:
+    *   current user is going to be used for `dcld` service to run as
+    *   current user is in sudoers list
+*   if it's not acceptable for your case please consult a less automated guide [Running Genesis Node](running-genesis-node.md)
+
 This command:
 
 *   generates a new key entry for a node admin account
 *   generates `genesis.json` file with the following entries:
     *   a genesis account for the key entry above with `Trustee` and `NodeAdmin` roles
     *   a genesis txn that makes the local node a validator
 *   configures and starts the node
-*   if `-u` option is specified the command will set a custom user the `dcld` service is executed as
 
-Outputs:
+### Validator Node
 
-*   `*.dclkey.json` file in the current directory with node admint key data (address, public key, mnemonic)
-*   standard output:
-    *   genesis file `$HOME/.dcld/config/genesis.json`
+This part describes how to configure a validator node and add it to the existing network.
 
-**Notes**
+The following steps automates a set of instructions that you can find in [Running Validator Node](running-validator-node.md) document
 
-    * by default the command will try to setup a systemd `dcld` service to start under `ubuntu` user
-    * if needed you may consier to specify a different service user `-u user`
+#### Create a key for a node admin
 
-### Validator Node
+```bash
+dclcli keys add <key-name> 2>&1 | tee -a <key-name>.dclkey.data
+```
 
-This part describes how to configure a validator node and add it to the existing network.
+*Notes*
 
-The following steps automates a set of instructions that you can find in [Running Validator Node](running-validator-node.md) document
+*   it's improtant to keep the generated data (especially a mnemonic that allows to recover the key) in a safe place
+*   note address and public key - they would be needed for a ledger account creation
 
 #### Setup a node
 
 Run
 
 ```bash
-$ ./run_dcl_node -c <chain-id> [-u <user>] <node-name>
+$ ./run_dcl_node -c <chain-id> <node-name>
 ```
 
+*Notes*
+
+*   the script assumes that:
+    *   current user is going to be used for `dcld` service to run as
+    *   current user is in sudoers list
+*   if it's not acceptable for your case please consult a less automated guide [Running Validator Node](running-validator-node.md)
+
 This command:
 
 *   generates a new key entry for a node admin account
```

Comment:
It doesn't generate any keys anymore, does it?

Reply:
yes, missed place, thanks